google-services/Location/ActivityRecognition/ActivityRecognition/packages.config

```diff
@@ -1,9 +1,60 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="monoandroid50" />
-  <package id="Xamarin.Android.Arch.Core.Common" version="1.0.0" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Arch.Lifecycle.Common" version="1.0.1" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Arch.Lifecycle.Runtime" version="1.0.0" targetFramework="monoandroid80" />
+  <package id="GoogleGson" version="2.6.1.0" targetFramework="monoandroid80" />

```

Comment:
Do we need this GoogleGson reference? I don't see it being used anywhere. Also presumably that means we can remove all of the additional seemingly .Net standard packages below here which are new additions?